### DISCUSSION TITLE
Influxdb

### DISCUSSION THREAD

**Discussion by rubenatorum** [alexjustesen/speedtest-tracker#149] [Q&A] [answered]
Hi, I would like to know how to configure the data to go to Influxdb. I have tried several ways and it does not export the data. I don't know how to specify the tags

**Comment by rubenatorum** [comment]
![Speedtracker_influxdb](https://user-images.githubusercontent.com/76797479/201636624-8f2cec57-ffef-4128-834a-a119054153ca.png)
This is my configuration

**Comment by alexjustesen** [comment, marked as answer]
This ended up being a gap in the docs, there is now an InfluxDB2 docs page that should help you out. Should you need any additional details please post below. https://docs.speedtest-tracker.dev/settings/data-platforms/influxdb2

**Reply by rubenatorum** [reply]
Hi, I still can't get it to work and send the data to Influxdb. I have changed all the parameters but it doesn't work. I have created a new bucket but it doesn't work either. I don't know what I'm doing wrong.

**Reply by alexjustesen** [reply]
It's possible the latency bug is causing the issue. I'll be back home in a couple days and will fix that.

**Reply by alexdelprete** [reply]
> Hi, I still can't get it to work and send the data to Influxdb. I have changed all the parameters but it doesn't work. I have created a new bucket but it doesn't work either. I don't know what I'm doing wrong.

Try changing the speedtest server id you are testing, just to be sure that the latency bug is not the issue. Unfortunately not all servers provide the same functionalities. Alex will prevent missing latency info from servers breaking influxdb export functionality.

**Reply by rubenatorum** [reply]
Thanks for your answers. The system works perfectly, the problem is that it does not send the data to my Influxdb. I have configured it in several ways and no data appears in Influxdb.

**Reply by rubenatorum** [reply]
Thank you very much, I have found the solution. The problem was mine. The token was not working properly. Sorry, it was my configuration problem. Congratulations for the work.
